### PR TITLE
JP Manage: Add the Sites Dashboard layout to support the Site Preview Pane

### PR DESCRIPTION
**Pull request by cleacos**
Fixes: https://github.com/Automattic/jetpack-manage/issues/291

## Proposed Changes

image

image

## Testing Instructions

*

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

File: client/jetpack-cloud/sections/agency-dashboard/sites-overview/sites-dataviews/index.tsx [hunk 1]
```diff
@@ -43,129 +55,138 @@ const SitesDataViews = ( {
...
+				id: 'plugins',
+				header: 'PLUGINS',
+				getValue: () => 'Plugins status',
+				render: ( { item }: { item: SiteData } ) => renderField( 'plugin', item ),
+				enableHiding: false,
+				enableSorting: false,
+			},
+			{
+				id: 'favorite',
+				//header: <Icon className="site-table__favorite-icon" size={ 24 } icon={ starFilled } />,
```
Comment (coder-karen) [inline client/jetpack-cloud/sections/agency-dashboard/sites-overview/sites-dataviews/index.tsx]: Why is this and the Actions header commented out?

**Review by coder-karen — COMMENTED** [review]
Just to double check, the intention of this PR is just to add the site preview pane as a shell, and not for example address any styling issues related to it?

File: client/jetpack-cloud/sections/agency-dashboard/sites-overview/sites-dataviews/index.tsx (hunk 1 above)
Comment (cleacos) [reply client/jetpack-cloud/sections/agency-dashboard/sites-overview/sites-dataviews/index.tsx]: I forgot to uncomment it. I checked how columns look without those headers (favorites and actions),  and then asked designers about it.

image

**VS**

image

**Comment by cleacos** [comment]
> Just to double check, the intention of this PR is just to add the site preview pane as a shell, and not for example address any styling issues related to it?

That is. But I add some styles to it to see the transition. We will have the proper stylish task for it. But If you see something weird, you can leave any feedback.

**Review by coder-karen — APPROVED** [review]
Adding an approval in order to have this merged to unblock other work, and acknowledging this is just an initial shell.

**Review by coder-karen — APPROVED** [review]

**Pull request merged**